[2024-08-12T16:31:11.120+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-12T16:31:11.128+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2024-08-11T00:00:00+00:00 [queued]>
[2024-08-12T16:31:11.129+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-08-12T16:31:11.130+0000] {taskinstance.py:1363} INFO - Starting attempt 4 of 4
[2024-08-12T16:31:11.130+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-08-12T16:31:11.140+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-08-11 00:00:00+00:00
[2024-08-12T16:31:11.147+0000] {standard_task_runner.py:55} INFO - Started process 19394 to run task
[2024-08-12T16:31:11.152+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2024-08-11T00:00:00+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp8_kri_b9']
[2024-08-12T16:31:11.155+0000] {standard_task_runner.py:83} INFO - Job 32: Subtask create_table
[2024-08-12T16:31:11.228+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2024-08-11T00:00:00+00:00 [running]> on host 7b166f9b76f0
[2024-08-12T16:31:11.294+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-08-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-08-11T00:00:00+00:00
[2024-08-12T16:31:11.306+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-08-12T16:31:11.312+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
            firstname  TEXT NOT NULL,
            lastname TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL
            );
        , parameters: None
[2024-08-12T16:31:11.314+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-08-12T16:31:11.325+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240811T000000, start_date=20240812T163111, end_date=20240812T163111
[2024-08-12T16:31:11.369+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-08-12T16:31:11.398+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
